### PR TITLE
Updated Umbraco UIBuilder 14.0.0-alpha1 docs

### PR DESCRIPTION
**Pull request by acoumb**
## Description

Updated docs for `Umbraco.UIBuilder 14.0.0-alpha1`.

## Type of suggestion

* [ ] Typo/grammar fix
* [x] Updated outdated content
* [ ] New content
* [x] Updates related to a new version
* [ ] Other

## Product & version (if relevant)

Umbraco UIBuilder 14.0.0-alpha1

## Deadline (if relevant)

Monday, September 2nd 2024

### CODE REVIEW

File: 14/umbraco-ui-builder/release-notes.md [hunk 1]
```diff
@@ -16,86 +16,13 @@ If you are upgrading to a new major version, check the breaking changes in the [
...
-
-* Fix built-in export/import actions errors [#84](https://github.com/umbraco/Umbraco.UIBuilder.Issues/issues/84)
-* Update list view `editorState` to support integration with `Contentment Data Types` [#83](https://github.com/umbraco/Umbraco.UIBuilder.Issues/issues/83)
-
-#### [13.0.0](https://github.com/umbraco/Umbraco.UIBuilder.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F13+label%3Arelease%2F13.0.0) (December 13th 2023)
-
-* Upgraded to run again Umbraco v13 and .NET 8
-* Upgraded all 3rd party dependencies
-* Nullable references
+You can read more about the new Backoffice [here](../umbraco-cms/extending/customize-the-editing-experience.md).
```
Comment (sofietoft) [inline 14/umbraco-ui-builder/release-notes.md:25]:
```suggestion
You can read more about the new Backoffice [in the Umbraco CMS documentation](https://docs.umbraco.com/umbraco-cms/extending/customize-the-editing-experience).
```

**Review by sofietoft — COMMENTED** [review]
I've added a single suggestion @acoumb ! 💪 

All these articles that you're adding a note to - will they be updated eventually?
Do you have them on a list somewhere to go through?

I'm asking because we should keep track of these notes somehow.
Over the past weeks, I've found articles that had been updated but still had the note on them 🙈 Makes it a little confusing, both for us, but also for our users.

**Comment by acoumb** [comment]
Hi @sofietoft ,

The sections I've added comments to are to be updated eventually when major version `14.0.0` will go out. New screenshots will need to be included as well. 

For virtual trees, child collections and related collections, these are not covered by the alpha release (virtual trees are dependent on updates in the CMS as well).

Thanks,
Adrian

**Comment by sofietoft** [comment]
Thanks for clearing that up, @acoumb ! 💪 

Let me know if you need help from our team regarding any of that.
I'll make sure this one is merged.

**Pull request merged**